modules/cluster-logging-manual-rollout-rolling.adoc

```diff
@@ -26,63 +26,39 @@ To perform a rolling cluster restart:
 $ oc project openshift-logging
 ----
 
-. Use the following command to extract the CA certificate from Elasticsearch and write to the *_admin-ca_* file:
+. Perform a shard synced flush using the {product-title} link:https://github.com/openshift/origin-aggregated-logging/tree/master/elasticsearch#es_util[*es_util*] tool to ensure there are no pending operations waiting to be written to disk prior to shutting down:
 +
 [source,terminal]
 ----
-$ oc extract secret/elasticsearch --to=. --keys=admin-ca
-----
-+
-[source,terminal]
-----
-admin-ca
-----
-
-. Perform a shard synced flush to ensure there are no pending operations waiting to be written to disk prior to shutting down:
-+
-[source,terminal]
-----
-$ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- curl -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key -XPOST 'https://localhost:9200/_flush/synced'
+$ oc exec <any_es_pod_in_the_cluster> -c elasticsearch -- es_util -s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key -XPOST 'https://localhost:9200/_flush/synced'
```

Comment:
We needn't specify the key when using es_util?
oc exec $es_pod -c  elasticsearch -- es_util  'https://localhost:9200/_flush/synced' -XPOST

Reply:
the second half of this should be `es_util --query="_flush/synced" -XPOST`

Reply:
@anpingli @ewolinetz Do we need the _-s --cacert /etc/elasticsearch/secret/admin-ca --cert /etc/elasticsearch/secret/admin-cert --key /etc/elasticsearch/secret/admin-key_? 
Doesn't seem to make a difference when I tried each.